ui-core/src/styles/inputs/datePicker.css

```diff
@@ -13,17 +13,25 @@
   }
 
   .calendar-picker {
-    position: absolute;
-    width: fit-content;
-    border-radius: 0.5rem;
-    box-shadow:
-      0 0.813rem 0.938rem -0.438rem rgba(31, 27, 23, 0.1),
-      0 0.188rem 0.375rem rgba(31, 27, 23, 0.2);
     display: flex;
     flex-direction: row;
     align-items: center;
     justify-content: center;
-    @apply bg-white-100;
+
+    .close-button {
+      position: absolute;
+      top: 16px;
+      right: 16px;
+      color: theme('colors.grey.50');
+
+      &:hover {
+        color: theme('colors.grey.90');
+      }
+
+      &:active {
+        color: theme('colors.grey.20');
+      }
+    }
```

Comment:
No need for that bit of code since it's already managed in the modal's css, right?
```suggestion

```